src/templates/ja/2019/base.html

```diff
@@ -108,9 +108,9 @@
     <p>— <em><a href="{{ url_for('contributors', year=year, lang=lang, _anchor='rviscomi') }}">Rick Viscomi</a>、Web Almanacの作成者。</em></p>
 {% endblock %}
 
-{% block appendix %}Appendix{% endblock %}
+{% block appendix %}付属資料{% endblock %}
 
-{% block ebook_download %}Download the entire {{ year }} Web Almanac in PDF format{% endblock %}
+{% block ebook_download %} {{ year }} Web AlmanacのPDFファイル{% endblock %}をダウンロードする
```

Comment:
This must start with `{% block ebook_download %}` and end with `{% endblock %}`
```suggestion
{% block ebook_download %}{{ year }} Web AlmanacのPDFファイルをダウンロードする{% endblock %}
```

Reply:
OK, I misstake and fixed.